sys/include/net/gnrc/ipv6/nib/pl.h

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_ipv6_nib_pl    Prefix list
+ * @ingroup     net_gnrc_ipv6_nib
+ * @brief       Prefix list component of neighbor information base
+ * @{
+ *
+ * @file
+ * @brief   Prefix list defintions
+ *
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+#ifndef NET_GNRC_IPV6_NIB_PL_H
+#define NET_GNRC_IPV6_NIB_PL_H
+
+#include <stdint.h>
+
+#include "net/ipv6/addr.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Prefix list entry view on NIB
+ */
+typedef struct {
+    ipv6_addr_t pfx;        /**< prefix */
+    uint16_t pfx_len;       /**< length of gnrc_ipv6_nib_pl_t::pfx in bits */
+    uint16_t iface;         /**< interface gnrc_ipv6_nib_pl_t::pfx is assigned
+                             *   to */
+    uint32_t valid_until;   /**< timestamp (in ms) until which the prefix is
+                                 valid */
+    uint32_t pref_until;    /**< timestamp (in ms) until which the prefix is
+                                 preferred */
+} gnrc_ipv6_nib_pl_t;
+
+/**
+ * @brief   Adds (or updates) prefix to NIB
+ *
+ * @pre `(pfx != NULL)`
+ *
+ * @param[in] iface         Interface @p pfx is valid on.
+ * @param[in] pfx           The prefix. May not be a link-local prefix or a
+ *                          multicast address and its first @p pfx_len bits
+ *                          may not be 0.
+ * @param[in] pfx_len       Length of @p pfx in bits.
+ *                          Condition 0 < @p pfx_len > 0 must hold.
```

Comment:
that's a very weird condition (:

Reply:
Its not weird... it's just a weird way to say `pfx_len > 0`... Anyway, will remove one of the ends ^^